src/core/shape/2d_primitives.js

```diff
@@ -168,6 +169,7 @@ p5.prototype._normalizeArcAngles = function(
 p5.prototype.arc = function(x, y, w, h, start, stop, mode, detail) {
   p5._validateParameters('arc', arguments);
 
+
```

Comment:
thanks @BerfinA! can you remove this extra empty line? then this pull request should be good to merge!